connectors/youtube.js

```diff
@@ -124,7 +144,19 @@ function setupMaterialPlayer() {
 	};
 
 	Connector.isStateChangeAllowed = function() {
-		return $('.videoAdUi').length === 0;
+		if ($('.videoAdUi').length > 0) {
+			return false;
+		}
+
+		const params = new URLSearchParams(document.location.search.substring(1));
+		const videoId = params.get('v');
```

Comment:
Yes, it is. Will change now.

(BTW, `Util.getYoutubeVideoIdFromUrl` is using regex for extracting ID. Isn't this way of getting `v` parameter a bit more reliable?)